python/PIK_maya_loader/scripts/PIKmenu/Rigging/ngskintools2/datas.json

```diff
@@ -0,0 +1,5 @@
+{
+    "scriptName" : "ngSkinTools2",
+    "annotation" : "ngSkinTools is a skinning plugin for Autodesk Maya, introducing new concepts to character skinning such as layers, any-pose-mirroring, enhanced paint brushes, true smoothing, and more.",
+    "scriptType" : "Python"
```

Comment:
The `scriptType` seems to be useless since you can autodetect if this is python or mel by looking at the `main.**` file extension.